internal/primitive/transform/function/strings_functions.go

```diff
@@ -140,3 +140,23 @@ var CapitalizeWord = function{
 		return string(rs), nil
 	},
 }
+
+var SplitFromStart = function{
+	name:      "SPLIT_FROM_START",
+	fixedArgs: []common.Type{common.String, common.Int},
+	fn: func(args []interface{}) (interface{}, error) {
+		value, _ := args[0].(string)
+		splitPosition, _ := args[1].(int)
+		if len(value) < 2 {
+			return []string{value}, nil
+		}
```

Comment:
The split function returns an array whose length is 2 but here returns an array length is 1, the results are inconsistent, so I think it doesn't need to add these codes, what about you?

Reply:
Really appreciate your feedback, and pointing to this. I agree.
Question, what should be returned by the function if the input string length is 1 or zero?

Reply:
The split function returns an array whose length must be 2, if the position is gte length of the value, return []string{value, ""}

Reply:
Good morning Team! @Michaelg22 @xdlbdy
Appreciate all your feedback. I was in a rush and made a mistake - I started requesting for your clarifications in two different places. As a result I got a little confused on several points. How about we try to consolidate the info here?

1. For the user the string starts with index (position) 1 or with index (position) 0?
- The issue description suggests that indexing for the user starts from 1. If we look at the "hello", it has 5 letters, so "h" is 1 for the user. 
![image](https://user-images.githubusercontent.com/56049635/223050628-dab611da-1582-4423-a237-3ea6f6fcea6d.png)
- Michael's recent comment suggests 0-based indexing. Thank made me want to double check.

2. Case: value == "". What should be returned?
3. Case: value == "h". What should be returned?
4. Case: value == "hello", split position >= length of the string. What should be returned?
5. Case: value == "hello", split position < the first element in the string. What should be returned?

Please let me know if some points don's make sense. Thank you for your support!

Reply:
1. For the user the string starts with index (position) 1 or with index (position) 0? The position uses 1 as started, only the index uses 0 as started.
the following question all follows rule returns an array whose length must be 2
2.  Case: value == "". It return []string("","")
3.  Case: value == "h". It return []string("h","") 
4. Case: value == "hello", split position >= length of the string. It return []string("hello","") 
5. Case: value == "hello", split position < the first element in the string. Do you mean the position <= 0?  It returns an error

Reply:
what about your idea? thank you.

Reply:
Sounds good to me. Let me try to implement this?

Reply:
Back merged updates from main, resolved conflicts locally and pushed to the same branch as this PR. 
Is that ok? Or should I make a new PR on a new branch?